.gitignore

```diff
@@ -5,5 +5,7 @@ public
 dist
 yarn-error.log
 coverage
+env
+.vscode
```

Comment:
👍 If anything, I'd include a few more, but obviously that's out of scope for this, so no need to.

I _think_ we could safely ignore anything matching `*.log` or beginning with a dot - I don't know of a time we need to commit dotfiles.

Reply:
I agree, I will add a few more... I've been getting .tmp files knocking around too which should really be ignored.